test/helpers/time.js

```diff
@@ -23,7 +24,17 @@ module.exports = {
 // TODO: deprecate the old version in favor of this one
 module.exports.bigint = {
   clock: mapObject(module.exports.clock, fn => () => fn().then(BigInt)),
-  clockFromReceipt: mapObject(module.exports.clockFromReceipt, fn => receipt => fn(receipt).then(BigInt)),
+  clockFromReceipt: {
+    blocknumber: txPromisse =>
+      Promise.resolve(txPromisse)
+        .then(tx => tx.blockNumber)
+        .then(BigInt),
+    timestamp: txPromisse =>
+      Promise.resolve(txPromisse)
+        .then(tx => ethers.provider.getBlock(tx.blockNumber))
+        .then(block => block.timestamp)
+        .then(BigInt),
```

Comment:
These aren't needed since the transaction response already contains `blockNumber`. I'm removing this and replacing where it fails